DevOps.Util.DotNet/HelixServer.cs

```diff
@@ -0,0 +1,104 @@
+﻿#nullable enable
+
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace DevOps.Util.DotNet
+{
+    public class HelixServer : BaseServer
+    {
+        public HelixServer()
+            : base((string?)null) { }
+
+        private HelixRequestBuilder GetBuilder(string apiPath) => new HelixRequestBuilder(apiPath);
+
+        public async ValueTask GetHelixPayloads(string jobId, List<string> workItems, string downloadDir)
+        {
+            if (!Path.IsPathFullyQualified(downloadDir))
+            {
+                downloadDir = Path.Combine(Environment.CurrentDirectory, downloadDir);
+            }
+
+            JobDetails jobDetails = await GetJsonAsync<JobDetails>(GetBuilder($"/jobs/{jobId}/details/"));
+            string? jobListFile = jobDetails.JobList;
+
+            if (string.IsNullOrEmpty(jobListFile))
+            {
+                Console.WriteLine($"Couldn't find job list for job {jobId}");
+                return;
+            }
+
+            using MemoryStream memoryStream = await DownloadFileAsync(jobListFile);
```

Comment:
Use `ConfigureAwait(false)` on all `await` calls in this project because it's a library.

Reply:
Makes sense.